static/js/stream_muting.js

```diff
@@ -21,7 +21,7 @@ export function update_is_muted(sub, value) {
             saved_ypos = message_viewport.scrollTop();
         } else if (
             message_lists.home === message_lists.current &&
-            message_lists.current.selected_row().offset() !== null
+            message_lists.current.selected_row().offset() !== undefined
```

Comment:
Why is this change needed?

Reply:
Actually, It was causing an error saying cannot read properties of undefined. I will change this to original code.